src/db/emotestore.ts

```diff
@@ -36,44 +37,51 @@ export class DbEmoteStore {
 	}
 
 	public async get(puppetId: number, roomId: string | null, emoteId: string): Promise<IEmoteStoreEntry | null> {
+		const stopTimer = this.db.latency.startTimer(this.labels("select"));
```

Comment:
Are these things any significant performance hits if metrics are disabled?

Reply:
I haven't noticed a significant performance hit but i also didn't run it through a profiler. The underlying function uses `process.hrtime` and calculates the diff, so there might be a tiny memory penalty if a lot of timers run at the same time. I couldn't get more metrics on it, because, well...

I suppose it's fine, i haven't noticed any slowdown